packages/website/components/Button/AddTokenButton.tsx

```diff
@@ -1,29 +1,67 @@
+import {
+  ELDFELL_ADD_ETHEREUM_CHAIN,
+  ELDFELL_CONFIG,
+  GRIMSVOTN_ADD_ETHEREUM_CHAIN,
+  GRIMSVOTN_CONFIG,
+  SEPOLIA_ADD_ETHEREUM_CHAIN,
+  SEPOLIA_CONFIG,
+} from "../../domain/chain";
+
 import { ethereumRequest } from "../../utils/ethereumRequest";
 
+type ConnectButtonProps = {
+  network:
+  | typeof SEPOLIA_CONFIG.names.shortName
+  | typeof ELDFELL_CONFIG.names.shortName
+  | typeof GRIMSVOTN_CONFIG.names.shortName;
+};
+
+const chainMap = {
```

Comment:
we shouldn't initialize constants here if we need this mapping we should expose it out of the domain folder